tests/roxctl/bats-tests/local/deployment-bundles-psps.bats

```diff
@@ -32,32 +32,32 @@ central_bundle_psp_disabled() {
     shift
     local bundle_dir="${out_dir}/bundle-central-${RANDOM}-${RANDOM}-${RANDOM}"
     roxctl-release central generate "${cluster_type}" pvc "$@" --output-dir="${bundle_dir}"
-    run grep -rq "kind: PodSecurityPolicy" "${bundle_dir}"
+    run grep -rq "kind: PodSecurityPolicy" "${bundle_dir}/central"
```

Comment:
Same here. Bug in the old testing code.

Reply:
How about extracting this to a separated bugfix PR?